RecoPixelVertexing/PixelTrackFitting/test/BuildFile.xml

```diff
@@ -14,6 +14,7 @@
 <use name="cuda"/>
 <use name="cuda-api-wrappers"/>
 <bin file="testEigenGPU.cu" name="testEigenGPU_t">
+  <flags CUDA_FLAGS="--expt-relaxed-constexpr -fmad=false -ftz=false -prec-div=true -prec-sqrt=true"/>
```

Comment:
why `-fmad=false` ?

Reply:
That was a list of flags to try and have a simpler assert in the code, but apparently, they did not work and I implemented the assert that went into the code. I can remove them.